server/src/lib/logger.ts

```diff
@@ -15,7 +15,6 @@ interface MessageFields {
   pid: number;
   msg: string;
   time: string;
-  version: number;
```

Comment:
Right, I took it out because it representend the DB version which doesn't really exist anymore (maybe the config key name should also be renamed). But I'm all for including the commit hash here.